packages/form-js-viewer/src/render/components/form-fields/IFrame.js

```diff
@@ -37,10 +34,16 @@ export function IFrame(props) {
 
   const evaluatedLabel = useSingleLineTemplateEvaluation(label, { debug: true });
 
-  const { formId } = useContext(FormContext);
+  const [ sandbox, allow ] = useSecurityAttributesMap(security);
+  const [ iframeRefresh, setIframeRefresh ] = useState(0);
+
+  // forces re-render of iframe when sandbox or allow attributes change, as browsers do not do it automatically
+  useEffect(() => {
+    setIframeRefresh(count => count + 1);
+  }, [ sandbox, allow ]);
```

Comment:
Kind of hacky, but I can't think of an option myself right now

Reply:
I mean I can also ref the iframe, then call the refresh manually via script, it's a little more code, it's equally hacky, so let's go for the virtual DOM hack instead of the regular DOM one.